doc/users-guide/users-guide-pktio.adoc

```diff
@@ -390,10 +390,21 @@ transmitted:
  * contain at least as many data bytes after L3/L4 offsets as the headers
  * indicate. Other data bytes of the packet are ignored for the checksum
  * insertion.
+ *
+ * Packet Tx timestamp insertion may be enabled or disabled.
```

Comment:
Talking about timestamp insertion is misleading since the timestamp is not inserted into the packet (like checksum is) but just reported to the application later. There are probably also other things in the API text that need some improvement but I think these can be postponed until the API design itself is clear.